core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -355,6 +355,18 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def getBillingProjectSpendConfiguration(billingProjectName: RawlsBillingProjectName): Future[Option[BillingProjectSpendConfiguration]] = {
+    requireProjectAction(billingProjectName, SamBillingProjectActions.alterSpendReportConfiguration) {
+      dataSource.inTransaction { dataAccess =>
+        dataAccess.rawlsBillingProjectQuery.load(billingProjectName).map {
+          case Some(RawlsBillingProject(_, _, _, _, _, _, _, _, Some(spendReportDataset), Some(spendReportTable), Some(spendReportDatasetGoogleProject))) => Option(BillingProjectSpendConfiguration(spendReportDatasetGoogleProject, spendReportDataset))
+          case Some(_) => None
+          case None => throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"Billing project ${billingProjectName.value} could not be found"))
```

Comment:
For an error, since this is called `BillingProjectSpendConfiguration`, maybe `"Billing Project ${billingProjectName.value}'s Spend Configuration could not be found"`?

Reply:
This wording was deliberately chosen because this is really just an unlikely edge case that the user could theoretically hit and it would only occur if the entire billing project was missing. That would only happen if the project resource somehow existed in Sam but had been deleted from Rawls, and in that case I think this wording is useful. The case for the spend configuration not being found is the `Some(_) => None` above it, which is still a successful case